{"when":1702924981,"controller":"rcc.user","event":"run","detail":"robot","comment":"task run"}
{"when":1702924981,"controller":"rcc.user","event":"processes","detail":"updated","comment":"count from 0 to 1 ... map[17804:python.exe]"}
{"when":1702924983,"controller":"rcc.user","event":"robot exit","detail":"From rcc point of view, \"actual main robot run\" was FAILURE, reason: exit status 1","comment":"rcc point of view"}
{"when":1702924983,"controller":"rcc.user","event":"processes","detail":"final","comment":"count: 0"}
{"when":1702924984,"controller":"rcc.user","event":"stop","detail":"robot","comment":"done"}
